README.md

```diff
@@ -116,5 +131,53 @@ New `wasm-code` binary file will be generated in the `runtime` folder.
 2. Build `wasm module` by executing `yarn run asbuild`
 3. Execute `yarn run test`
 
+### 6. Run the node with WASM code
+1. Go to `./node-template`
+2. Copy `wasm-code` generated earlier from `../runtime`
+3. Install `jq` with `apt-get` or `brew`
+4. Run `bash cr-custom-spec.sh` to format `customSpec.json`
```

Comment:
We could simplify the instructions by telling users to copy the wasm-code and place it in the `code` property manually, instead of installing `jq` or running script